components/TestCaseEditor.tsx

```diff
@@ -69,42 +71,52 @@ export default function TestCaseEditor() {
 
   return (
     <>
-      <Grid style={{ height: 'calc(100vh - 120px)' }}>
+      <Grid className={classes.gridContainer}>
         <Grid.Col span={3} className={classes.panel}>
           <PatientCreationPanel />
         </Grid.Col>
-        <Grid.Col
-          span={3}
-          className={classes.panel}
-          sx={theme => ({
-            backgroundColor: theme.colors.gray[1],
-            borderLeft: `2px solid ${theme.colors.gray[4]}`,
-            borderRight: `2px solid ${theme.colors.gray[4]}`
-          })}
-        >
+        <Grid.Col span={3} className={cx(classes.panel, classes.resourcePanelRoot)}>
           <ResourcePanel />
         </Grid.Col>
-        <Grid.Col span={6} className={classes.calculation} sx={theme => ({ backgroundColor: theme.colors.gray[1] })}>
-          <Box h={50}>
+        <Grid.Col span={6} className={cx(classes.calculation, classes.highlightPanelRoot)}>
+          <Stack h={50}>
             {selectedPatient ? (
-              <Grid justify="space-between">
-                <Grid.Col span={4}>
-                  Patient Calculation: {getPatientNameString(currentPatients[selectedPatient].patient)}
-                </Grid.Col>
-                <LoadingSpinner />
-              </Grid>
+              <Group position="right">
+                <Center pr={20}>
+                  {isCalculationLoading ? (
+                    <Center>
+                      <Loader size={40} />
+                      <Text italic color="dimmed">
+                        Calculating...
+                      </Text>
+                    </Center>
+                  ) : (
+                    <Center>
+                      <CircleCheck color="green" size={40} />
```

Comment:
Really minor suggestion here but this circle check looks pretty large compared to the size of the text - I would probably bump the size down to like 28 or 32-ish